lm_eval/evaluator.py

```diff
@@ -46,8 +47,8 @@ def simple_evaluate(
     :param model_args: Optional[str]
         String arguments for each model class, see LM.create_from_arg_string.
         Ignored if `model` argument is a LM object.
-    :param tasks: list[Union[str, Task]]
-        List of task names or Task objects. Task objects will be taken to have name task.EVAL_HARNESS_NAME if defined and type(task).__name__ otherwise.
+    :param tasks: list[Task]
```

Comment:
I think we should really not change the public interface in such a way--a lot of users rely on the `evaluate()` and `simple_evaluate()` APIs to use the library externally and would be affected by it changing on them.